src/features/courses/course-controller.ts

```diff
@@ -4313,7 +4429,11 @@ You should be able to reply to the student's email address (${options.student.em
             }
             return unit;
         });
-        logger.info(`Import Course Archive complete ${new Date().getTime() - startTime} ${new Date()}`);
+        logger.info(`Import Course Archive complete ${JSON.stringify({
+            rendererSavePGFileRequests,
+            rendererSaveAssetRequests,
+            rendererAccessibleRequests,
+        })} ${new Date().getTime() - startTime} ${new Date()}`);
```

Comment:
nit: Node should have `performance.now`. I don't know that it provides much accuracy benefit over the date object, though.

Reply:
Yeah agreed, eventually I plan to take this out anyway (or I guess I could demote to debug statements)